### DISCUSSION TITLE
Move Master Harvester or  Full Node to another Server

### DISCUSSION THREAD

**Discussion by retrominer** [Chia-Network/chia-blockchain#4518] [Support]
Hi all,

Is there a way to move my Master Harvester or  Full Node to another Server ?

Thanks for your support

kind regards,
RetroMiner

**Comment by lvcivs** [comment]
Haven't tried it, but I guess it could work like this? 

1. Shut down your full node on server1
2. Install chia on server2 and add keys
3. copy the `~/.chia` dir from server1 to server2
4. start full node on server2

**Comment by retrominer** [comment]
thanks for reply

But that now work. I got messages keys are problems.

**Reply by retrominer** [reply]
Only a full restore of my OS works

**Comment by BrandtH22** [comment]
@retrominer , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - make sure to perform step 2 recommended by Ivcivs as that is where you would add the keyset which would resolve the key errors.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
